FSharpBuild.Directory.Build.props

```diff
@@ -80,7 +80,7 @@
   </PropertyGroup>
 
   <!-- SDK targets override -->
-  <PropertyGroup Condition="'$(Configuration)' != 'Proto' AND '$(DisableCompilerRedirection)'!='true' AND Exists('$(ProtoOutputPath)')">
+  <PropertyGroup Condition="'$(Configuration)' != 'Proto' AND '$(DisableCompilerRedirection)'!='true'">
```

Comment:
@brettfo This `Exists` was meaning that failure to build or find Proto wasn't detected.  Do you see any problem with removing it?

Reply:
I don't remember why I added that, but it's probably a good idea to remove it.